internal/weldr/api.go

```diff
@@ -1805,6 +1806,70 @@ func (api *API) blueprintsDiffHandler(writer http.ResponseWriter, request *http.
 	common.PanicOnError(err)
 }
 
+// blueprintsChangeHandler returns a specific change to a blueprint
+func (api *API) blueprintsChangeHandler(writer http.ResponseWriter, request *http.Request, params httprouter.Params) {
+	if !verifyRequestVersion(writer, params, 1) {
+		return
+	}
+	name := params.ByName("blueprint")
+	if !verifyStringsWithRegex(writer, []string{name}, ValidBlueprintName) {
+		return
+	}
+
+	commit := params.ByName("commit")
+	if !verifyStringsWithRegex(writer, []string{commit}, ValidBlueprintName) {
```

Comment:
Isn't `commit` supposed to have a different regex than `ValidBlueprintName`?

Reply:
It could, but in practice it doesn't matter since the commit is just a string, and if it doesn't match it will fail. We're using SHA1 for commit strings right now, but could use anything including uuid's including a `-`.